zerver/lib/message.py

```diff
@@ -529,12 +529,7 @@ def render_markdown(message: Message,
             if user_id in message_user_ids:
                 possible_words.update(set(words))
 
-    if message is None:
-        # If we don't have a message, then we are in the compose preview
-        # codepath, so we know we are dealing with a human.
-        sent_by_bot = False
-    else:
-        sent_by_bot = get_user_profile_by_id(message.sender_id).is_bot
+    sent_by_bot = get_user_profile_by_id(message.sender_id).is_bot
```

Comment:
I agree with the reasoning here, but we should also clean up the `if message is not None` conditional above, too.